Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Rasta Clothes Fixes

## Changelog:
- Fixed an issue where the Rasta cloak clipped with clothes.
- Fixed an issue where troll empresses wore nothing.


# How to test:
> Fixed an issue where the Rasta cloak clipped with clothes.

Can be noticed only in the [Portrait Editor](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/Portrait-Editor).
- Open the Portrait Editor.
- Open the Gene list. Select `clothes`.
- Open the Sub group list. Select `zandalari_imperial`
- Open the Gene list again. Select `cloaks`.
- Open the Sub group list. Select `rastakhan_cloak`.
- Open the 3D View. See if it clips.

> Fixed an issue where troll empresses wore nothing.

Check how spouses of the troll emperors look.
